benchmark-cache.sh

```
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}=== Git Proxy Hybrid Cache Benchmark ===${NC}"
+echo ""
+
+# Configuration
+PROXY_URL="http://localhost:8000"
+GITHUB_REPO="${1:-fabiovincenzi/open-webui}"
```

Perhaps we could also throw an error if the user didn't provide their own repo in the arguments rather than defaulting to your fork?

---

Oh, I tought i removed all the references of my github, I just realized I didn't commit those